### DISCUSSION TITLE
how to add admin role to a user ?

### DISCUSSION THREAD

**Discussion by huanlirui** [logto-io/logto#2108] [Q&A]
The details are here #2107

**Comment by charIeszhao** [comment]
Well, thanks for asking.

To answer your question directly, we can't assign admin role to a normal use in current version. And we only have 1 admin user.

In order to support it, we need to implement the RBAC (role based access control) feature, and it's on our roadmap and expected to release in Q4.

Feel free to join our [Discord channel](https://discord.gg/C4c7RPxD) to get in touch us and all the community members.

**Reply by huanlirui** [reply]
Thank you for your answer
If I want to use the API of the logto users table in my own business system, can I only log in to the admin user for operations?

**Reply by gao-sun** [reply]
@charIeszhao
